### ISSUE TITLE
Multiverse-Inventories Conflict with Gamerule keepInventory

### ISSUE THREAD

**Issue by mrretrus** [cybertiger/Bukkit-Instances#40]
Gamerules are being inherited fine which is a good start- Seperately, multiverse Inventories are properly being carried to other worlds (I enter the instance with my stuff, whatever I leave the instance with is what I bring out of it) However- with the Gamerule keepinventory set to true- When I die the inventories get swapped out in some strange order I can quite put my finger on. Lets say I die with 3 dirt, I dont drop it. I respawn in the instance and collect some glass- and die again, but I respawn witout the glass. Then I leave the instance back to the root world that has the entrance to the instance. I die, and respawn with the glass now-

So basically the inventories get mixed up whenever gamerule keepinventory is set to true in the parent world. This is not a problem with multiverse itself as the parent world properly maintains the inventories across all worlds- only when i enter an instance and die does it start to happen
